src/modules/RemoveTask.js

```diff
@@ -0,0 +1,36 @@
+import UpdateIndex from './UpdateIndex.js';
+import { GetTasks } from './ReadTasks.js';
+import StoreTask from './UpdateTasks.js';
+
+let taskToDelete = [];
+
+export const RemoveFromLocal = (tasks, taskId) => tasks.filter((task) => taskId !== task.id);
+
+export const SelectTask = (idTask) => {
+  const mainLi = document.getElementById(idTask);
+  const checkBox = document.getElementById(`check-${idTask}`);
+  const addedInputTask = mainLi.querySelector('input[type=text]');
+  checkBox.addEventListener('change', function select() {
+    if (this.checked) {
+      addedInputTask.classList.add('text-decoration-line-through');
+      taskToDelete.push(idTask);
+      return;
+    }
+    addedInputTask.classList.remove('text-decoration-line-through');
+    taskToDelete = taskToDelete.filter(((task) => idTask !== task));
+  });
+};
```

Comment:
- [ ] When the checkbox is checked the `completed` property of the item in the local storage should be updated to `true`. Kindly implement that. Right now, only a strike-through is added to the task, but the property is not updated in the local storage.

Reply:
Thanks for your comment. I am going to implement it